src/potentials.h

```diff
@@ -645,6 +713,44 @@ TEST_CASE("[Faunus] FunctorPotential") {
     CHECK(u(c, c, r * 0.99) == pc::infty);
 }
 
+TEST_CASE("[Faunus] Dipole-dipole interactions") {
+    using doctest::Approx;
+
+    json j = R"({ "atomlist" : [
+                 {"A": { "mu":[3.0,0.0,0.0] }},
+                 {"B": { "mu":[0.0,3.0,0.0] }} ]})"_json;
+
+    atoms = j["atomlist"].get<decltype(atoms)>();
+
+    FunctorPotential u = R"(
+                {
+                  "default": [
+                    { "dipoledipole" : {"epsr": 1.0, "type": "plain", "cutoff":20} }
+                  ]
+                 }
+                )"_json;
+
+    DipoleDipole dipoledipole = R"({ "dipoledipole": {"epsr": 1.0, "type": "plain", "cutoff":20} } )"_json;
+
+    Particle a = atoms[0];
+    Particle b = atoms[1];
+    Point r = {2, 0, 0};
+    CHECK(u(a, a, r) == Approx(dipoledipole(a, a, r))); // interaction between two parallell dipoles, directed parallell to their seperation
```

Comment:
this checks if FunctorPotential gives same results as dipoledipole. Add test that compares with precalculated values? (probably for some less idealized dipole orientations)